src/main/java/PieChartView.java

```diff
@@ -53,7 +55,7 @@ private void drawPieChartWithLabels(Graphics g) {
         int centerXChart = panelWidth / 2; // Exact X center
         int centerYChart = panelHeight / 2; // Exact Y center
 
-        Font broadwayFont = new Font("Arial", Font.BOLD, 20);
+        Font broadwayFont = new Font("Arial", Font.BOLD, 10);
```

Comment:
The font size was changed from 20 to 10, likely to avoid text overlap. However, this could make text less legible, especially for small pie slices.